core/src/main/scala/org/apache/spark/rdd/MapPartitionsRDD.scala

```diff
@@ -54,4 +58,12 @@ private[spark] class MapPartitionsRDD[U: ClassTag, T: ClassTag](
 
   @transient protected lazy override val isBarrier_ : Boolean =
     isFromBarrier || dependencies.exists(_.rdd.isBarrier())
+
+  override private[spark] def computingRandomLevel = {
```

Comment:
computingRandomLevel -> computeOrderSensitivity
(suffix to match what `RandomLevel` gets renamed to)